src/parsers/manifest/metaplaylist/metaplaylist_parser.ts

```diff
@@ -293,13 +293,16 @@ function createManifest(
   }
 
   const time = performance.now();
+  const isLastPeriodKnown = !isDynamic ||
+                            mplData.pollInterval === undefined;
```

Comment:
Shouldn't we also check that the last inner Manifest's period is known?

Reply:
Good point !